osx/ka-lite-remover.sh

```diff
@@ -209,7 +221,13 @@ echo "  KALITE_HOME with value $KALITE_HOME"
 if [ $IS_PREINSTALL == false ]; then
     echo "         "
     echo -n "Do you wish to uninstall KA-Lite? (Yes/No) "
-    read uninstall
+    # Check if the first input argument in the script was passed.
+    if [ "$1" != "" ]; then
```

Comment:
Change this to specifically accept a valued argument, not just any value, before you allow auto-uninstall.